Getting-Started-with-Akka-http-signature.md

```diff
@@ -15,7 +15,7 @@ The user can also run the rww-play web server and manipulate the access control
 
 ## The Key Pair
 
-The Akka-Http-Signature library uses asymmetrical cryptography. As detailed in the [Public-Key cryptography wikipedia page](https://en.wikipedia.org/wiki/Public-key_cryptography), this is a cryptographic system that uses pairs of private and public keys. The library uses them to form Digital Signatures. In such a system public keys can be looked up by anyone and private keys are only known by their owner. Communication is achieved the following way : 
+The Akka-Http-Signature library uses what is known as asymmetrical cryptography. As detailed in the [Public-Key cryptography wikipedia page](https://en.wikipedia.org/wiki/Public-key_cryptography), this is a cryptographic system that uses pairs of private and public keys. The library uses them to form Digital Signatures. In such a system public keys can be looked up by anyone and private keys are only known by their owner. Communication is achieved the following way : 
 1. The sender sends a message with a header that is signed by his own private key 
 2. The server then receives the message and uses the sender's public key to verify that sender's identity
```

Comment:
add that the server discovers the public key through using the `keyId` argument passed in the the message sent by the client. The rww-play server works with http and https:// urls and fetches the public key that way.